aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -647,87 +942,125 @@ def export_tree(
-    with folder.open('data.json', mode='w') as fhandle:
-        # fhandle.write(json.dumps(data, cls=UUIDEncoder))
-        fhandle.write(json.dumps(data))
+    :param folder: a temporary folder to build the archive before compression.
 
-    # Turn sets into lists to be able to export them as JSON metadata.
-    for entity, entity_set in entities_starting_set.items():
-        entities_starting_set[entity] = list(entity_set)
+    :param allowed_licenses: List or function. If a list, then checks whether all licenses of Data nodes are in the
+        list. If a function, then calls function for licenses of Data nodes expecting True if license is allowed, False
+        otherwise.
 
-    metadata = {
-        'aiida_version': get_version(),
-        'export_version': EXPORT_VERSION,
-        'all_fields_info': all_fields_info,
-        'unique_identifiers': unique_identifiers,
-        'export_parameters': {
-            'graph_traversal_rules': graph_traversal_rules,
-            'entities_starting_set': entities_starting_set,
-            'include_comments': include_comments,
-            'include_logs': include_logs
-        }
-    }
+    :param forbidden_licenses: List or function. If a list, then checks whether all licenses of Data nodes are in the
+        list. If a function, then calls function for licenses of Data nodes expecting True if license is allowed, False
+        otherwise.
 
-    with folder.open('metadata.json', 'w') as fhandle:
-        fhandle.write(json.dumps(metadata))
+    :param silent: suppress console prints and progress bar.
 
-    EXPORT_LOGGER.debug('ADDING REPOSITORY FILES TO EXPORT ARCHIVE...')
+    :param include_comments: In-/exclude export of comments for given node(s) in ``entities``.
+        Default: True, *include* comments in export (as well as relevant users).
 
-    # If there are no nodes, there are no repository files to store
-    if all_node_pks:
-        all_node_uuids = {node_pk_2_uuid_mapping[_] for _ in all_node_pks}
+    :param include_logs: In-/exclude export of logs for given node(s) in ``entities``.
+        Default: True, *include* logs in export.
 
-        progress_bar = get_progress_bar(total=len(all_node_uuids), disable=silent)
-        pbar_base_str = 'Exporting repository - '
+    :param traversal_rules: graph traversal rules. See :const:`aiida.common.links.GraphTraversalRules` what rule names
+        are toggleable and what the defaults are.
 
-        for uuid in all_node_uuids:
-            sharded_uuid = export_shard_uuid(uuid)
+    :raises `~aiida.tools.importexport.common.exceptions.ArchiveExportError`: if there are any internal errors when
+        exporting.
+    :raises `~aiida.common.exceptions.LicensingException`: if any node is licensed under forbidden license.
+    """
+    # pylint: disable=unused-argument
+    warnings.warn('function is deprecated, use `export` instead', AiidaDeprecationWarning)  # pylint: disable=no-member
```

Comment:
```suggestion
    warnings.warn('function is deprecated and will be removed in AiiDA v2.0.0, use `export` instead', AiidaDeprecationWarning)  # pylint: disable=no-member
```

Reply:
fixed in b689f5e6986c15ab18ad225e5659f1ef3bf8f9c0